### ISSUE TITLE
Native Color Picker no longer available

### ISSUE THREAD

**Issue by jamiehunterdesign** [hananils/kirby-colors#11]
The Mac native color picker does not appear when clicking the color swatch, as it did before.

**Comment by nilshoerrmann** [comment]
Thanks for the report. 

What do you refer to with "as it did before"? 
Before updating Kirby, the plugin, the browser?
Which browser are you using?

**Comment by jamiehunterdesign** [comment]
I'm not sure exactly what changed as the plugin was implemented in my project for a while and only just now have I noticed the change. However, it seems to have only impacted Chrome and Safari.

In Chrome, previously when clicking the color swatch it would bring up my Mac's native color picker allowing me to sample a color from anywhere on the screen. Now, when clicking the color swatch, a different kind of color picker pops up attached to the field and I can no longer use the color sampler.

Firefox still shows the native color picker. Safari does neither of the things described above.
